index.html

```diff
@@ -4,30 +4,32 @@
 <div class="container my-4">
   <div class="row">
     <div class="col-md-6">
-      <a href="https://monitor.opensuse.org/icingaweb2/">
+      <a href="https://alerts.infra.opensuse.org/">
         <h2 id="monitoring">
           {% include monitor.svg %}
-          Monitoring page
+          Alerting page
         </h2>
       </a>
       <p>
-        Get detailed monitoring information from our infrastructure. 
-        This page requires valid log in data (<a href="https://en.opensuse.org/openSUSE:Heroes" target="_blank">openSUSE heroes</a> account).
+        Detailed monitoring alerts from our infrastructure. 
+        This page is currently only accessible internally through the <a href="https://en.opensuse.org/openSUSE:Heroes" target="_blank">openSUSE Heroes</a> VPN.
       </p>
       <p>
-        We recently switched from 
-         <a href="https://github.com/Icinga/icinga-core" title="Icinga Github Repo" target="_blank">Icinga</a>/
-         <a href="https://checkmk.com/" title="Check MK" target="_blank">Check_MK</a>
-          to an 
+        We transitioned from an
          <a href="https://github.com/Icinga/icinga2" title="Icinga2 Github Repo" target="_blank">Icinga2</a>
-          setup with 
+          setup with the
          <a href="https://github.com/Icinga/icingaweb2" title="IcingaWeb2 Github Repo" target="_blank">IcingaWeb</a>
-          frontend. The checks are distributed via our internal Salt-Master.
+          frontend to a
+         <a href="https://prometheus.io" title="Prometheus homepage" target="_blank">Prometheus</a>
+          based monitoring solution. The new configuration is entirely managed using
+         <a href="https://docs.saltproject.io/en/latest/topics/about_salt_project.html" title="Salt documentation" target="_blank">Salt</a>
+         and can be found in our 
+         <a href="https://code.opensuse.org/heroes/salt/tree/production" title="Heroes Salt Git Repository" target="_blank">Salt Git repository</a>.
       </p>
       <p>
-        In general, we use packages from our preferred <a href="https://www.opensuse.org">distribution</a>
-        , but enhance our setup with some additional packages from the 
-        <a href="https://build.opensuse.org/project/show/server:monitoring" title="server:monitoring Repo" target="_blank">server:monitoring</a> 
+        The setup uses packages from our preferred <a href="https://www.opensuse.org">distribution</a>
+        , and is enhanced with custom packages gathered in the
```

Comment:
Nitpick: please move the comma directly after the link to avoid whitespace.

Reply:
Ok!